src/analyses/threadId.ml

```diff
@@ -156,12 +156,13 @@ struct
     let non_uniques = List.filter_map (fun (a,b) -> if not (Thread.is_unique a) then Some a else None) tids in
     let uc = List.length uniques in
     let nc = List.length non_uniques in
-    Printf.printf "Encountered number of thread IDs (unique): %i (%i)\n" (uc+nc) uc;
-    Printf.printf "unique: ";
-    List.iter (fun tid -> Printf.printf " %s " (Thread.show tid)) uniques;
-    Printf.printf "\nnon-unique: ";
-    List.iter (fun tid -> Printf.printf " %s " (Thread.show tid)) non_uniques;
-    Printf.printf "\n"
+    Logs.debug "Encountered number of thread IDs (unique): %i (%i)" (uc+nc) uc;
```

Comment:
These should be printed independently of the logging level, as it already is controlled by `dbg.print_tids`. Also, as this not some random debugging information, maybe these should be issued via the messaging system?